### PR TITLE
[stable/coredns] Version bump to 1.6.1

### PR DESCRIPTION
**Pull request by mrueg**
https://github.com/coredns/coredns/blob/master/notes/coredns-1.6.0.md

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/coredns/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: coredns
-version: 1.5.6
-appVersion: 1.5.2
+version: 1.5.7
```
Comment (cpanato) [inline stable/coredns/Chart.yaml]: please bump to 
```suggestion
version: 1.6.0
```

Comment (mrueg) [reply stable/coredns/Chart.yaml]: Done, thanks for the quick review!

**Review by batazor — APPROVED** [review]

**Comment by mrueg** [comment]
@cpanato can you take a look again? Thanks!

**Pull request merged**